gulpfile.js

```diff
@@ -269,6 +268,25 @@ function coveralls() { // 2nd arg is a dependency: 'test' must be finished
     .pipe(shell('cat build/coverage/lcov.info | node_modules/coveralls/bin/coveralls.js'));
 }
 
+gulp.task('watch', function () {
```

Comment:
this gulp task is redundant and can be removed; there is already a `watch` function declared earlier in the file (see [here]( https://github.com/prebid/Prebid.js/blob/master/gulpfile.js#L106)).

Reply:
yes, removed